explainers/hidden-content-explainer.md

```diff
@@ -54,7 +53,7 @@ section. The net effect is that the user is able to use find-in-page or link
 navigation to find content in collapsed sections -- something that is not
 currently possible.
 
-### Primary Use Case: collapsed searchable sections
+## Primary use case: collapsed searchable sections
```

Comment:
Should I implement an example, or just add another heading that describes how it could work and show an example like github's long comment threads (which wouldn't actually work with this since clicking "load more" comments makes an actual network request)?

Reply:
I think most virtualized concatenated article cases issue network requests, since if you have the text already available for search then there is little need to virtualize the content

Reply:
hmm chris what do you think about mine and vlads points?

Reply:
I think it's ok to skip this thing I suggested.